GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -404,6 +404,12 @@ private void CreateSubmoduleNodes(SubmoduleInfoResult result, GitModule threadMo
                 foreach (var submoduleInfo in result.AllSubmodules)
                 {
                     string superPath = GetSubmoduleSuperPath(submoduleInfo.Path);
+                    if (string.IsNullOrEmpty(superPath))
```

Comment:
This will hide the problem for the user. Can this be displayed somehow in the list instead?
(Maybe todo)